examples/advanced/src/example.tsx

```diff
@@ -564,16 +564,34 @@ export default class Example extends React.Component<
                 this.state.modeConfig.turfOptions.units) ||
               'kilometers'
             }
-            onChange={(event) =>
-              this.setState({ modeConfig: { turfOptions: { units: event.target.value } } })
-            }
+            onChange={(event) => {
+              const modeConfig = {
+                ...this.state.modeConfig,
+                turfOptions: { units: event.target.value },
```

Comment:
Is this a new option? Generally we should document options, and write tests for them.

Reply:
No. This param is an old one and it's already covered in docs https://github.com/uber/nebula.gl/blob/master/docs/api-reference/modes/overview.md